packages/website/src/pages/foundations/Spacing.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+
+import {PageLayout} from '../../building-blocs/PageLayout';
+
+const code = `
+    import * as React from 'react';
+
+    // red is margin
+    // blue is padding

```

Comment:
I would either add a comment for `highlight-padding` and `highlight-margin` to say they are only for the demo or add a way to specify a class for the sandbox container

```suggestion
    // red is margin
    // blue is padding
    // highlight-padding and highlight-margin are only available on the demo website
```